.github/PULL_REQUEST_TEMPLATE/release.md

```
@@ -0,0 +1,135 @@
+## Deployment
+
+**Deployment steps:**
+1. 
+2. 
+3. 
+
+**Rollback plan:**
+1. 
+2. 
+
+**Configuration changes:**
```

markdownlint-cli2 (0.22.1)

[warning] Heading style
Expected: atx; Actual: setext

(MD003, heading-style)